examples/tv-casting-app/android/App/app/src/main/jni/com/matter/casting/support/DataProvider.java

```diff
@@ -33,4 +33,15 @@ protected T _get() {
   }
 
   public abstract T get();
+
+  /**
+   * Must be implemented in the CommissionableData DataProvider if the Commissioner-Generated
+   * passcode commissioning flow is going to be used. In this flow, the setup passcode is generated
+   * by the CastingPlayer and entered by the user in the tv-casting-app CX. Once it is obtained,
+   * this function should be called with the Commissioner-Generated passcode to update the
+   * CommissionableData DataProvider in AppParameters. The client is also responsible for calling
+   * CastingApp::updateAndroidChipPlatformWithCommissionableData() to update the data provider set
+   * which was previously set in the AppParameters and AndroidChipPlatform at initialization time.
+   */
+  default void updateCommissionableDataSetupPasscode(long setupPasscode, int discriminator) {}
```

Comment:
We don't need to add this method here - see comments in ConnectionExampleFragment

Reply:
ok